hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/balance/AccountBalanceFileParser.java

```diff
@@ -76,7 +76,7 @@ public AccountBalanceFileParser(MeterRegistry meterRegistry, BalanceParserProper
             maxDelayExpression = "#{@balanceParserProperties.getRetry().getMaxBackoff().toMillis()}",
             multiplierExpression = "#{@balanceParserProperties.getRetry().getMultiplier()}"),
             maxAttemptsExpression = "#{@balanceParserProperties.getRetry().getMaxAttempts()}")
-    @Transactional
+    @Transactional(timeoutString = "#{@balanceParserProperties.getTransactionTimeout()}")
```

Comment:
With my other suggestion:
```suggestion
    @Transactional(timeoutString = "#{@balanceParserProperties.getTransactionTimeout().toSeconds()}")
```

Reply:
updated